cli/src/main/resources/nls/Help.properties

```diff
@@ -10,18 +10,18 @@ cmd.build=Runs a build job with given arguments.
 cmd.build.detail=The `build` commandlet is an abstraction of build systems like https://maven.apache.org/, https://gradle.org/, https://yarnpkg.com/, https://www.npmjs.com/, etc.\nIt will auto-detect your build-system (via existence of files like `pom.xml`, `package.json`, etc.). According to this detection, it will simply delegate to the according commandlet of the specific build system. If that build-system is not yet available it will be downloaded and installed automatically.\nSo `ide build` allows users to build any project without bothering about the build-system. Further specific build options can be configured per project. This makes `ide build` a universal part of every _definition of done_. Before pushing your changes, please always run the following command to verify the build:\n\n 'ide build'\n\nYou may also supply additional arguments as `ide build «args»`. This will simply delegate these arguments to the detected build command (e.g. call `mvn «args»`).
 cmd.build.val.args=Build arguments to be used when running a build job.
 cmd.complete=Internal commandlet for bash auto-completion.
-cmd.complete.detail=TODO complete
+cmd.complete.detail=Run 'ide complete <expression>' to activate the non-interactive autocompletion, replace <expression> with the expression you want to autocomplete.\nF.e type: 'ide complete in' to get 'install' and 'intellij' suggestions.
 cmd.create=Create a new IDEasy project.
-cmd.create.detail=TODO create
+cmd.create.detail=Run 'ide create <project_name>' to create a new IDEasy project in the projects directory, replace <project_name> with the name of your project (it will become the directory name).
```

Comment:
DRY?
```
Usage: ide [option]* create <project> [<settingsRepository>]
Create a new IDEasy project.
```

Reply:
I've removed the repetition of the project_name now.

Reply:
I do not see this update.
Maybe instead of 
```
Run 'ide create <project_name> <settings_repository>' to create a new IDEasy project in the projects directory using your custom project settings...
```
just write
```
Create a new IDEasy project in the projects directory (IDE_ROOT) using your custom project settings...
```

Ah, and by the way:
> You can also omit the <settings_repository> to use the default project settings.

If you want to have the default settings, you need to provide `-` as `<settings_repository>`.
If you omit it, you will get an interactive question for the git settings URL.

Reply:
I've adjusted it now as you've suggested.